### PR TITLE
Add instructions for adding a moderator

### PR DESCRIPTION
**Pull request by castrojo**
Signed-off-by: Jorge O. Castro <jorgec@vmware.com>

### CODE REVIEW

File: communication/moderation.md [hunk 1]
```diff
@@ -220,6 +220,19 @@ umbrella.
 - [Zoom Guidelines](./zoom-guidelines.md)
 - [Moderators](./moderators.md#zoom)
 
+### Process for Adding a Moderator
+
+This is the workflow for adding a moderator. 
+
+- Moderator applies by filing an issue in the kubernetes/community repo
```
Comment (mrbobbytables) [inline communication/moderation.md]: It might be worth linking directly to the issue template for these 👍

File: communication/moderation.md [hunk 2]
```diff
@@ -220,6 +220,19 @@ umbrella.
...
+- Moderator get's 2 approvals from sponsors
+- Add the person to their respective moderation tool:
+  - Slack - Add them as a slack workspace admins (instructions pinned on #slack-admins-private): Invite them to #slack-admins-private, #slack-admins, #slack-log, #slack-reports and #slack-invites
+  - Discuss - Change their permission to moderator
+  - Google group - PR the person into the correct yaml file in kubernetes/k8s.io
```
Comment (mrbobbytables) [inline communication/moderation.md]: This is only true things that `@kubernetes.io` properties. k-dev etc are still managed out of band through the UI

File: communication/moderation.md [hunk 3]
```diff
@@ -220,6 +220,19 @@ umbrella.
...
+  - Ensure they add that they are an admin in their profile page on whatever service they are administrating
+- Add them to the moderators mailing list by PRing them into this file: https://github.com/kubernetes/k8s.io/blob/master/groups/groups.yaml
```
Comment (spiffxp) [inline communication/moderation.md]: In the PR description, include a link to the issue the moderator filed

File: communication/moderation.md [hunk 4]
```diff
@@ -220,6 +220,19 @@ umbrella.
 - [Zoom Guidelines](./zoom-guidelines.md)
 - [Moderators](./moderators.md#zoom)
 
+### Process for Adding a Moderator
+
+This is the workflow for adding a moderator. 
+
+- Moderator applies by filing an issue in the kubernetes/community repo
+- Moderator get's 2 approvals from sponsors
```
Comment (mrbobbytables) [inline communication/moderation.md]:
```suggestion
- Moderator gets 2 approvals from sponsors
```
nit -- might also be better worded as `Moderator gets approval from 2 current moderators`

File: communication/moderation.md [hunk 5]
```diff
@@ -244,3 +257,4 @@ guidelines:
 [Steering Committee]: https://git.k8s.io/steering
 [moderator request]: https://github.com/kubernetes/community/issues/new/choose
 [moderator template]: https://git.k8s.io/community/.github/ISSUE_TEMPLATE/moderator_application.md
+[filing an issue]: https://github.com/kubernetes/community/issues/new/choose
```
Comment (spiffxp) [inline communication/moderation.md]: I think this is what @mrbobbytables was asking for
```suggestion
[filing an issue]: https://github.com/kubernetes/community/issues/new?assignees=&labels=area%2Fcommunity-management%2C+sig%2Fcontributor-experience&template=moderator_application.md&title=REQUEST%3A+New+moderator+for+%3Cyour-GH-handle%3E+of+%3Ck8s+property%3E
```

Comment (mrbobbytables) [reply communication/moderation.md]: It was^^;;;

**Pull request merged**